guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+   width = 800
+   height = 348
+%}
+
+{% include tip/open.html color="blue" icon="info" label="Sending and receiving" %}
+
+If your app offers the user visibility into the channel management process, you might consider using the term "receive limit" in place of "inbound liquidity" and "send limit" in place of "outbound liquidity".
+
+{% include tip/close.html %}
+
+## Why is liquidity important?
+
+Liquidity directly impacts the user experience. If a user has no outbound liquidity, they can not send bitcoin over lightning. Conversely, if a user has no inbound liquidity, they can not receive bitcoin over lightning.
+
+Sourcing inbound liquidity is very important for onboarding a user to a lightning wallet. Imagine if you were brand new to bitcoin. You download your first wallet, a lightning wallet. Because there is no inbound liquidity, you can not receive any bitcoin. Would you understand what to do next? Would the wallet still seem useful to you?
+
+Helping the user to find inbound liquidity will remove friction from onboarding and make their overall experience with bitcoin easier.
+
+## Identifying different use cases
+
+Before going further, it's important to make a distinction between different use cases for lightning nodes. It is difficult to make firm categories for lightning nodes. However, we can categorize them very generally for our purposes within the Design Guide. Here are some very broad examples of different use cases.
+
+<div class="nodes -full-width" markdown="1">
+   <table>
+      <thead>
+         <tr>
+            <th>&nbsp;</th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-rachel.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-rachel@2x.png"
+                  alt-text = "Depiction of Rachel and a dedicated hardware lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Routing Node Operator
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-miguel.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-miguel@2x.png"
+                  alt-text = "Depiction of Miguel and a cloud lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Merchant
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-wagner.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-wagner@2x.png"
+                  alt-text = "Depiction of Wagner and a mobile phone lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Wallet User
```

Comment:
Merchants and routing node operators also use wallets, I'd simply call this 'Payments' and change the 'What is their incentive?' to "Wagner uses a mobile bitcoin wallet that supports lightning to send and receive payments.

Could also link to the DSP reference design as that section encompasses this use case.

Reply:
Agreed with changing the description to "Wagner uses...". Disagree with changing the name to "Payments".

Reply:
Updated in fbcd880